app/views/v2/events/_form.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= form_for @event, url: v2_events_path, html: { class: 'form-horizontal' } do |f| %>
+  <div class="control-group">
+    <%= label_tag :email_address, "Person's email address", :class => 'control-label' %>
+    <div class="controls">
+      <%= text_field_tag :email_address, nil, :class => 'text_field' %>
+    </div>
+  </div>
+  <div class="control-group">
+    <%= f.label :description, 'Event description', :class => 'control-label' %>
+    <div class="controls">
+      <%= f.text_area :description, :class => 'text_area' %>
+    </div>
+  </div>
+  <div class="control-group">
+    <%= label_tag :slot_length, 'Call length', :class => 'control-label' %>
+    <div class="controls">
+      <%= select_tag :slot_length, options_for_select([15,30,45,60].map {|o| "#{o.to_s} mins"}), :class => 'text_field' %>
+    </div>
+  </div>
+
+  <h2>Time window</h2>
+  <div class="control-group">
+    <%= label_tag :date, nil, :class => 'control-label' %>
+    <div class="controls">
+      <%= text_field_tag :date, nil, :class => 'text_field' %>
+    </div>
+  </div>
+
+  <div class="control-group">
+    <%= label_tag :start_time, nil, :class => 'control-label' %>
+    <div class="controls">
+      <%= select_tag :start_time, simple_time_select_options, :class => 'text_field' %>
+    </div>
+  </div>
+
+  <div class="control-group">
+    <%= label_tag :end_time, nil, :class => 'control-label' %>
+    <div class="controls">
+      <%= select_tag :end_time, simple_time_select_options, :class => 'text_field' %>
+    </div>
+  </div>
+
+  <div class="form-actions">
+    <%= f.submit 'Send invitation', :class => 'btn btn-primary' %>
+    <%= link_to t('.cancel', :default => t("helpers.links.cancel")),
+                people_path, :class => 'btn' %>
+  </div>
+<% end %>
```

Comment:
Good point, seems like the simplest solution to implement for now. And eventually it might be nice to have it more dynamic? With start and end times adapting to the call length, and end times adapting to the start time, and so on. Something to discuss on project weekly